gems/pending/util/postgres_admin.rb

```diff
@@ -12,6 +12,10 @@ def self.data_directory
     Pathname.new(ENV.fetch("APPLIANCE_PG_DATA"))
   end
 
+  def self.template_directory
+    Pathname.new(ENV.fetch("APPLIANCE_TEMPLATE_DIRECTORY"))
+  end
```

Comment:
Initially, I was going to make a ruby module on the system side that exposed all of those ENV variables through ruby methods that places like this could call.